test/common_distributed.py

```diff
@@ -91,6 +92,16 @@ def requires_mpi():
     )
 
 
+def skip_for_rocm(func):
```

Comment:
nit: I feel it's better to name it "skip_if_rocm" to be more consistent with the `skipIfRocm` `decorator`

Reply:
I agree. Will update.